src/main/scala/interplay/PlayBuildBase.scala

```diff
@@ -111,6 +116,33 @@ object PlayBuildBase extends AutoPlugin {
   )
 }
 
+private object PlaySbtBuildBase extends AutoPlugin {
+
+  override def trigger = noTrigger
+  override def requires = PlayBuildBase
+
+  private def choose[T](scalaBinVersion: String)(forScala210: T, forScala212: T) = CrossVersion.partialVersion(scalaBinVersion) match {
+    case Some((2, 12)) => forScala212
+    case _ => forScala210
+  }
+
+  override def projectSettings = Seq(
+    crossScalaVersions := Seq(ScalaVersions.scala210, ScalaVersions.scala212),
+    sbtVersion in pluginCrossBuild := choose(scalaBinaryVersion.value)(
+      forScala210 = SbtVersions.sbt013,
+      forScala212 = SbtVersions.sbt10
+    ),
+    javacOptions in compile ++= choose(scalaBinaryVersion.value)(
+      forScala210 = Seq("-source", "1.6", "-target", "1.6"),
```

Comment:
This is probably fine (since it's the same as what it was before), but I'm pretty sure Play's sbt plugin depends on Java 7 APIs (file watch service for example).

Reply:
I did like this to keep the same behavior we already have.